seed/views/v3/cycles.py

```diff
@@ -0,0 +1,83 @@
+# !/usr/bin/env python
+# encoding: utf-8
+"""
+:copyright (c) 2014 - 2020, The Regents of the University of California,
+through Lawrence Berkeley National Laboratory (subject to receipt of any
+required approvals from the U.S. Department of Energy) and contributors.
+All rights reserved.  # NOQA
+:authors Paul Munday<paul@paulmunday.net> Fable Turas <fable@raintechpdx.com>
+"""
+from seed.models import Cycle
+
+from seed.serializers.cycles import CycleSerializer
+from seed.utils.viewsets import SEEDOrgNoPatchOrOrgCreateModelViewSet
+from seed.utils.api_schema import AutoSchemaHelper
+
+
+class CycleSchema(AutoSchemaHelper):
+    def __init__(self, *args):
+        super().__init__(*args)
+
+        self.manual_fields = {
```

Comment:
Seems like all of these actually take the organization ID field. Without it, the currently selected organization (in the top right) has it's ID inserted in the params automatically. I'm not quite sure how that happens.

I'd vote to keep them all consistent, either all endpoints take the org ID explicitly from the user or all endpoints get the org ID from the currently selected organization. I'm leaning towards the explicit option (user provides ID), but definitely let me know what you think.